action.yml

```
@@ -58,6 +58,13 @@ inputs:
     description: "If true a MySQL 5.7 will be set up"
     required: false
     default: true
+  mysql-schema:
+    description: "MySQL database schema to use. Can be Mysql or Mariadb"
+    options:
+      - Mysql
+      - Mariadb
+    required: false
+    default: 'Mysql'
```

I'd go with a generic db-provider here as it will allow us to run different database engines as well, not being forks or feature-matching engines with MySQL.

Then I would have the values all lowercase as then we don't need to worry about correct capitalisation of the names.

```suggestion
  db-provider:
    description: "Database provider to use. Can be 'mysql' or 'mariadb'."
    options:
      - mysql
      - mariadb
    required: false
    default: 'mysql'
```

---

Alternatively db-engine or mysql-engine, but I don't think schema is correct here, that has a different meaning imho.